### ISSUE TITLE
Windows: GCS 4.16.0 Fails to Open on Java 8

### ISSUE THREAD

**Issue by Hubok** [richardwilkes/gcs#201]
Discovered this one by accident when setting up my dev environment and JDKs with GCS open. On Windows, GCS 4.16.0 (from the main MSI installer) is dependent on javadir\bin\zip.dll. No GCS_LOG entries were made during testing.

That file exists in that same path on OpenJDK 15 EA and AdoptOpenJDK 9-14.
On AdoptOpenJDK 8, it exists in javadir\jre\bin.
On Java SE 8 JRE, it does exist in the normal place, but I get: "The procedure entry point JVM_GetLastErrorString could not be located in the dynamic link library [dir]\zip.dll."

The alt MSI installer version starts up with no version of Java installed at all, while the main MSI requires Java 9+.

**Comment by richardwilkes** [comment]
Interesting. You're saying the **installer** crashes? I've not had any reports of that, only that the copy of GCS installed with the main installer on Windows doesn't work for some people. In those cases, setting GCS_LOG won't help, because none of my code ever runs -- it dies on JVM startup. I've not been able to determine what is different on those systems from mine. Given that GCS bundles the JVM runtime that it uses, whether a system already has Java installed or not shouldn't be relevant.

**Comment by Hubok** [comment]
Let me rephrase what I said, I was very tired when I gave the initial report and made it very out of order.

- Installation works just fine regardless of what I do, I'm using the two MSIs you provide in the release.
- If I install from the main MSI, the program will not start unless I have Java 9+.
- If I install from the alt MSI, the program will run even if I do not have Java installed.
- If I have AdoptOpenJDK 8 with zip.dll in an unexpected place, it will say nothing and not open.
- If I have Java SE 8 JRE, I get the error launching it.

I've had my GCS_LOG env var set for a while, I just noted that failure to launch or errors launching did not write to that file.

**Comment by richardwilkes** [comment]
Ah, that clears things up a lot. Thanks. Clearly a bug in the unreleased JDK 15 packaging support. Too bad, as that code base fixes a number of Windows-specific issues, which is why I was trying to use it.

**Comment by Hubok** [comment]
I see that the 4.17.0 release only has a single MSI, and I've confirmed that on that version Java is not required to run it. Do you have any issues up for those issues that were fixed in the JDK 15 packager?

**Comment by richardwilkes** [comment]
I just reverted back to Java 14 for now. The things 15 fixed weren’t worth the hassle.
